File: Solution/UISolution/Original/src/app/leaves/Models/LeaveApplication.ts
export class LeaveApplication{
    constructor(
        public id:number,
        public employeeId:number,
        public applicationDate:string,
        public fromDate:string,
        public toDate:string,
        public status:string,
        public year:number,
        public leaveType:string
        ){}
}

File: Solution/WebSolution/Monolithic/Entities/LeaveMgmt/LeaveApplication.cs
namespace Transflower.TFLPortal.Entities.LeaveMgmt;

public class LeaveApplication
{
    public int Id { get; set; }
    public DateTime CreatedOn { get; set; }
    public DateTime FromDate { get; set; }
    public DateTime ToDate { get; set; } 
    public string? Status { get; set; }
    public string? LeaveType { get; set; }
    public int EmployeeId { get; set; }
}
